src/server.js

```diff
@@ -25,6 +25,7 @@ Server.prototype.listen = function(port, host) {
     var client = new Client(true);
     client._end = client.end;
     client.end = function end(endReason) {
+      endReason='{"text":"'+endReason+'"}';
```

Comment:
Yep, I just tried adding a 
```js

  setTimeout(function(){
    client.end("go away please");
  },15000);
```
at https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/examples/proxy/proxy.js#L84
and it just works :)